### ISSUE TITLE
Wanted to Add Security Camera Ai

### ISSUE THREAD

**Issue by Vaibhav-kesarwani** [abhisheks008/DL-Simplified#871]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Security Camera ai</br>
:red_circle: **Aim** : This web app will help you to detect the intruders </br>
:red_circle: **Dataset** : Object-detection(Human) - Kaggle</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Vaibhav Kesarwani
- GitHub Profile Link : https://github.com/Vaibhav-kesarwani
- Email ID : vaibhavkesarwani100@gmail.com
- Participant ID (if applicable):
- Approach for this Project : to Protect and have the best security system
- What is your participant role? (Mention the Open Source program)  GSSoC

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Can you elaborate more on this project?
@Vaibhav-kesarwani

**Comment by anshuman444** [comment]
can you like assign this to me

**Comment by abhisheks008** [comment]
Please fill up the details like this and comment again,
![image](https://github.com/user-attachments/assets/cfef1a9c-65dd-446e-9964-19bc702462cb)

@anshuman444

**Comment by anshuman444** [comment]
Full Name- Anshuman Dash
GitHub Profile Link - https://github.com/anshuman444
Email ID- anshumandash909@gmail.com
Approach For this Project - Identify and locate specific objects within the camera's field of view (e.g., people, vehicles, weapons).
Participant role -kwoc

**Comment by abhisheks008** [comment]
Assigned to you, @anshuman444

**Comment by anshuman444** [comment]
@abhisheks008 do you have any discord channel the project is almost ready and its working fine

**Comment by anshuman444** [comment]
@Vaibhav-kesarwani  WANTS TO CONTACT

**Comment by abhisheks008** [comment]
> [@abhisheks008](https://github.com/abhisheks008) do you have any discord channel the project is almost ready and its working fine

Yes, there is a discord channel for communication purposed for KWOC. Here you go, https://discord.gg/X3TTUP6 @anshuman444

**Comment by abhisheks008** [comment]
You have merged your pull request in your repository. You need to fork the Deep Learning Simplified repository first and create a pull request to the main branch, then only I can see your pull request.
@anshuman444
